server/src/main/java/io/unitycatalog/server/service/CoordinatedCommitsService.java

```diff
@@ -63,4 +68,38 @@ public HttpResponse commit(Commit commit) {
     }
     return HttpResponse.of(HttpStatus.OK);
   }
+
+  @Get("")
+  public HttpResponse getCommits(
+      @Param("table_id") String tableId,
+      @Param("table_uri") String tableUri,
```

Comment:
Can we validate tableUri matches the tableUri of existing entries in CommitDAO for this table?

Reply:
We don't store the table URI in the commit table. I don't think DB UC does that either